lib/src/main/java/io/cloudquery/plugin/Plugin.java

```diff
@@ -18,6 +18,7 @@ public abstract class Plugin {
   @NonNull protected final String name;
   @NonNull protected final String version;
   @Setter protected Logger logger;
+  @Setter protected String json_schema;
```

Comment:
`jsonSchema` i.e. camel case in Java

Reply:
4efb5febdc82b3a843b3594c55942b3f5b6d6f5d